interfaces/dbus/backend.go

```diff
@@ -74,10 +118,46 @@ func (b *Backend) Setup(snapInfo *snap.Info, opts interfaces.ConfinementOptions,
 	return nil
 }
 
+func (b *Backend) setupBusActivatedSessionServ(snapInfo *snap.Info, spec interfaces.Specification) error {
+	snapName := snapInfo.Name()
+
+	content := map[string]*osutil.FileState{}
+	for securityTag, serviceContent := range spec.(*Specification).SessionServices() {
+		fname := fmt.Sprintf("%s.service", securityTag)
```

Comment:
If not we can pipe the security tag through systemd-escape, which is maybe safe in any case when we generate a systemd unit name.